src/openvic-simulation/military/UnitInstance.cpp

```diff
@@ -0,0 +1,29 @@
+#include "UnitInstance.hpp"
+#include "military/UnitType.hpp"
+#include <vector>
```

Comment:
```suggestion
#include "UnitInstance.hpp"

#include <vector>

#include "openvic-simulation/military/UnitType.hpp"
```
Header arrangement + full path